abs.yaml

```diff
@@ -2,3 +2,9 @@
 build_parameters:
   - "--suppress-variables"
   - "--skip-existing"
+
+channels:
```

Comment:
will versions in this channel be overwritten ? i.e. will new commits after review result in updated packages or will it `--skip-existing`?

Reply:
Right now no, because `--skip-existing` is set, at least for this package.

Reply:
yeah sorry, to be clearer, you should remove skip-existing if you want the versions to be the ones corresponding to the last build on this PR - i.e., after everything has been reviewed

Reply:
This is fine as the version changed as well.

Reply:
ah you are going to remove the channels: stuff before merge and take the artifacts uploaded in the merge commit? I was thinking you would take the artifacts from anaconda.org and copy them across.